### ISSUE TITLE
Prev/Next-Aria-Labels are not applied for prev/next-buttons if slots aren't used

### ISSUE THREAD

**Issue by dploeger** [buefy/buefy#3028]
### Overview of the problem

**Buefy** version: [0.9.4]
**Vuejs** version: [2.6.12]
**OS/Browser**: Chrome 86.0.4240.183 on macOS 10.15.7

### Description

The properties `aria-next-label` and `aria-previous-label` are not applied to the default pagination buttons.

### Steps to reproduce

Add a pagination and set `aria-next-label` and `aria-previous-label` without using the next and previous-slots.

### Expected behavior

The previous and next buttons should have the given labels applied.

### Actual behavior

If no aria-page-label is provided, no labels will be set at all. If an aria-page-label is provided, that will be used and the target page will be appended.

https://codesandbox.io/s/buefy-table-example-forked-x86po?file=/src/App.vue

**Comment by dploeger** [comment]
Awesome, thanks a lot.
